src/App.js

```diff
@@ -12,6 +19,7 @@ import MockAPI from "./Mockman";
 import { ArchivedPage } from "./Pages/ArchivedPage/ArchivedPage";
 import { LabelPage } from "./Pages/LabelPage/LabelPage";
 import { LandingPage } from "./Pages/LandingPage/LandingPage";
+import { TrashPage } from "./Pages/TrashPage/TrashPage";
 
 function App() {
   const location = useLocation();
```

Comment:
Rather than finding the path from `location.pathname` create a layout component with sidebar and outlet and then wrap that layout component around `<Route element={<RequireAuth />}>`

Reply:
Sure will check this out